.eslintrc.json

```diff
@@ -20,6 +50,10 @@
         "ignoreDestructuring": true
       }
     ],
+    "arrow-parens": [
+      2,
+      "always"
+    ],
     "no-underscore-dangle": 0
```

Comment:
Why no-underscore-dangle?

Reply:
In fact, we are disabling no-underscore-dangle warning.

Reply:
Why?

Reply:
why should be blocked underscore?